gepetuto/__main__.py

```diff
@@ -55,17 +55,20 @@ def parse_args() -> argparse.Namespace:
 def main():
     """Run command."""
     args = parse_args()
+    tp_ids = list(range(100))
+    if args.tp_id:
+        tp_ids = args.tp_id
```

Comment:
Instead of defining a new variable, we could provide the default list to the `default` argument of `argparse.add_argument()`